src/main/java/edu/project1/GameState.java

```diff
@@ -0,0 +1,38 @@
+package edu.project1;
+
+/**
+ * The {@code GameState} enum represents the various states or outcomes that can occur during the execution
+ * of a word-guessing game. Each enum constant represents a different game state and provides a corresponding
+ * message that can be displayed to the player.
+ */
+public enum GameState {
+    START(String.format(Settings.ATTEMPTS_MESSAGE, Settings.MAX_ATTEMPTS_COUNT)
+        + String.format(Settings.EXIT_INFO_MESSAGE, Settings.EXIT_COMMAND)),
```

Comment:
Не нравится конкатенация тут, кажется более читаемо было бы объеденить `Settings.ATTEMPTS_MESSAGE` и `Settings.EXIT_INFO_MESSAGE` в одну строку.
```suggestion
    START(startMessage.formatted(Settings.MAX_ATTEMPTS_COUNT, Settings.EXIT_COMMAND)),
```

Reply:
Исправил.